public/locales/en/claim-status.json

```diff
@@ -0,0 +1,11 @@
+{
+  "pending-determination": {
+    "description": "We need to confirm your eligibility for benefits."
+  },
+  "base-pending": {
+    "description": "Your payments are pending further review."
+  },
+  "base-no-pending": {
+    "description": "There are no pending issues at this time."
+  }
+}
```

Comment:
@nicoleslaw: As a first pass at this, I'm trying out breaking out all the Claim Status scenarios into a `claim-status.json` file. Within the file, each scenario is a top level object. Let me know what you think about this approach! I'm very open to changing it.